accepted/0000-powershell-bins.md

```diff
@@ -54,3 +54,4 @@ The path to the bin script will have to be substituted into the right place.
 ## Unresolved Questions and Bikeshedding
 
 - impact of additional scripts on install times, node_modules size, etc.
+- default powershell permissions: will powershell users need to toggle some settings so this works without permissions errors? Can the errors be worked around? (This PR will need testing on a fresh Windows install)
```

Comment:
c.f. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-6

Short answer: it depends on the version of Windows. Newer installs default to `RemoteSigned`, which should be fine.

What should happen if the user's policy won't allow running scripts? Maybe don't create the `ps1` scripts on `install`?

Reply:
I may have spoken too soon. Seems like newer Windows Server defaults to `RemoteSigned`, but Windows 10 is still `Restricted`.

My tentative recommendation would be to shell out with something like this:

```powershell
powershell -NoProfile -Command Get-ExecutionPolicy
```

If it returns `Unrestricted` or `RemoteSigned`, `ps1` shims should work fine.

Reply:
I think it would be great if we could still find some solution that improves the experience for all those who have the execution policy configured - it's one of those first things you do as a dev when you setup a Windows machine, and many tools (e.g. Chocolatey package manager) don't work without it.

One thing to consider when shelling out is that PowerShell Core (v6) uses `pwsh` as the executable name. I don't know if they share the same execution policy, maybe? I guess it's stored in the registry somewhere.